Mail/Views/Thread List/ThreadListViewModel+Observation.swift

```diff
@@ -55,10 +55,8 @@ extension ThreadListViewModel {
 
                     DispatchQueue.main.sync {
                         self.filteredThreads = filteredThreads
-                        withAnimation(animateInitialThreadChanges ? .default : nil) {
-                            self.sections = newSections
-                        }
                     }
+                    sectionsSubject.send(newSections)
```

Comment:
Technically this is not the same as before as the call is now async and throttled but the I didn't see issues with my tests.